Formula/arx-libertatis.rb

```diff
@@ -25,6 +25,23 @@ class ArxLibertatis < Formula
   def install
     args = std_cmake_args
 
+    # The patches for these aren't straightforward to backport because of
+    # other changes; these minimal inreplaces get it building.
+    # HEAD is fine, and the next stable release will contain these changes.
+    if build.stable?
+      inreplace "src/platform/Time.cpp", "clock_t", "clockid_t"
+
+      # Version parsing is broken in the current stable; fixed upstream.
+      # This hardcodes the current version based on data from VERSION.
+      inreplace "src/core/Version.cpp.in" do |s|
```

Comment:
These version issues are what using the old CMake addressed, but hard-coding them temporarily instead of vendoring the "correct" CMake seems fine to me.